index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bezalell Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a class="name-container" href="logo">Bezalell</a>
+            <img class="union" src="images/Union.png" alt="">
+        </nav>
+    </header>
+    <!-- Homepage Of Portfolio -->
+    <section class="header">
+        <div>          
+            <h1 class="welcome">
+              Hey There.<br>I' am Bezalell
+            </h1>
+            <h2 class="job-desc">Im a Software Developer</h2>
+    
+            <p class="intro">I can help you build a product , feature or<br> website Look through some of my work <br> and experience! If you like what you see<br> and have a project you need coded,<br> 
+              don't hestiate to contact me.</p>
+          </div>
+          
+           <div>
+            <ul class="icons-container">
+              <li><a><img src="images/github.png" alt="github"/></a></li>
+              <li><a><img src="images/linkedin.png" alt="linkedin"/></a></li>
+              <li><a><img src="images/angel list.png" alt="angel-list"/></a></li>
+              <li><a><img src="images/twitter.png" alt="twitter"/></a></li>
+              <li><a><img src="images/medium.png" alt="medium"/></a></li>
+            </ul>           
+           
+    </section >
```

Comment:
- [OPTIONAL] As requested before I would like to recommend you change the style of the home page section because now it is not matching with the Figma template.

** Your design**

![Screenshot from 2023-03-01 07-34-17](https://user-images.githubusercontent.com/105117832/222141517-31a6267a-00c2-4ca9-8308-e4dee3fa3ec0.png)

**Figma**

![Screenshot from 2023-03-01 07-35-26](https://user-images.githubusercontent.com/105117832/222141562-635a8c0d-7539-4060-80db-25c1bcb52224.png)